docs/guides/Integration-Guides/erp-integration-guide/erp-integration-set-up-order-integration.md

```diff
@@ -3,68 +3,65 @@ title: "Set up order integration"
 
 ### Filter
 
-The Feed configuration filters what types of order updates actually generate events in the queue. This allows you to select the exact types of updates that are relevant for your integration. The Feed can be filtered by changes in status (`”type”: “FromWorkflow”`) or by any changes made to the order JSON document (`”type”: “FromOrders”`).
+The feed configuration filters the types of order updates that generate events in the queue. This allows you to select the exact types of updates relevant to your integration. The feed can be filtered by changes in status (`”type”: “FromWorkflow”`) or by any changes made to the order's JSON document (`”type”: “FromOrders”`).
 
-When filtering changes in status (`”type”: “FromWorkflow”`) for ERP integrations, `ready-for-handling` is probably the most important to monitor. `start-handling` and `handling` may also be relevant depending on your operation’s architecture. For marketplaces with external fulfillment, `cancelation-request` and `waiting-ffmt-authorization` may also be pertinent.
+When filtering changes in status (`”type”: “FromWorkflow”`) for ERP integrations, `ready-for-handling` is probably the most important to monitor. `start-handling` and `handling` may also be relevant depending on the architecture of your operation. For marketplaces with third-party fulfillment, `cancelation-request` and `waiting-ffmt-authorization` may also be important.
 
-The other filtering type (`”type”: “FromOrders”`) can also be used to filter changes in status or any other change made to an order, which provides a much more customizable option. You can, for example, filter orders that have been delivered or that have had items removed or added to by the store.
+The other filtering type (`”type”: “FromOrders”`) can also be used to filter changes in status or any other change made to an order, which provides a more customizable option. You can, for example, filter orders that have been delivered or had items removed or added to by the store.
 
 Learn more about how to configure filtering options in this [Feed API guide](https://developers.vtex.com/docs/guides/orders-feed).
 
-> The orders feed is unique to each `appKey` / `appToken` so that it can be configured and used by different applications with no interference between them.
+> The order feed is unique to each `appKey` / `appToken` so that it can be configured and used by different applications without interfering with each other.
+> ⚠️ The feed configuration does not validate the inserted values. Make sure the elements in the `filter.status` array follow the spelling of the [OMS workflow strings](https://help.vtex.com/en/tutorial/order-flow-and-status--tutorials_196?&utm_source=autocomplete) exactly.
```

Comment:
```suggestion
> The order feed is unique to each `appKey` / `appToken` so that it can be configured and used by different applications without interfering with each other.

>⚠️ The feed configuration does not validate the inserted values. Make sure the elements in the `filter.status` array follow the spelling of the [OMS workflow strings](https://help.vtex.com/en/tutorial/order-flow-and-status--tutorials_196) exactly.
```